app/panel/components/Subscription/SubscriptionThemes.jsx

```diff
@@ -22,7 +22,7 @@ import { RadioButtonGroup } from '../BuildingBlocks';
  * @memberOf SettingsComponents
 */
 const SubscriptionThemes = (props) => {
-	const themes = ['default', 'midnight-theme', 'palm-theme', 'leaf-theme'];
+	const themes = ['default', 'dark-blue-theme', 'palm-theme', 'leaf-theme'];
```

Comment:
won't this cause the theme to not load as it is served as `midnight-theme`by the account api?

Reply:
Yup, it will break older versions. Changed that guy back, and it looks like the ping parameter is tied to that theme name, so we're good on GH-1974 too